baseplate/core.py

```diff
@@ -523,6 +520,9 @@ def create(cls, authentication_context=None, loid_id=None,
         :param str session_id: (Optional) ID for the current session cookie.
 
         """
+        from .thrift.ttypes import Loid as TLoid
```

Comment:
could you put a comment above these explaining why so it's not cleaned up in a lint later?